packages/manager/modules/core/src/index.js

```diff
@@ -76,72 +78,74 @@ angular
   // Fix security issue: https://github.com/angular-translate/angular-translate/issues/1418
   .factory(
     'translateMissingTranslationHandler',
-    ($sanitize) => (translationId) => $sanitize(translationId),
+    /* @ngInject */ ($sanitize) => (translationId) => $sanitize(translationId),
+  )
+  .run(
+    /* @ngInject */ (tmhDynamicLocaleCache, tmhDynamicLocale, coreConfig) => {
+      const injectAngularLocale = (lang) =>
+        tmhDynamicLocaleCache.put(
+          lang,
+          angular.injector(['ngLocale']).get('$locale'),
+        );
+      const defaultLanguage = coreConfig.getUserLocale();
+      const angularLocale = kebabCase(defaultLanguage);
+
+      let angularLocalePromise;
+      switch (angularLocale) {
+        case 'de-de':
+          angularLocalePromise = import('angular-i18n/angular-locale_de-de.js');
+          break;
+        case 'en-gb':
+          angularLocalePromise = import('angular-i18n/angular-locale_en-gb.js');
+          break;
+        case 'en-us':
+          angularLocalePromise = import('angular-i18n/angular-locale_en-us.js');
+          break;
+        case 'es-es':
+          angularLocalePromise = import('angular-i18n/angular-locale_es-es.js');
+          break;
+        case 'fi-fi':
+          angularLocalePromise = import('angular-i18n/angular-locale_fi-fi.js');
```

Comment:
**suggestion:** Both `fi-fi` and `lt-lt` can be removed.

Relates to:  #4229 and #4365

Reply:
Will address it in another PR 👍

Reply:
Done in https://github.com/ovh/manager/pull/4598